internal/http/handler.go

```diff
@@ -668,23 +668,21 @@ func handleAssignPolicy(config *ServerConfig) http.HandlerFunc {
 			return
 		}
 
-		identity := kes.Identity(pathBase(r.URL.Path))
-		if identity.IsUnknown() {
+		var request Request
+		if err = json.NewDecoder(r.Body).Decode(&request); err != nil {
```

Comment:
Should it be possible for `json.NewDecoder(r.Body)`, that because a malformed `r.Body`, the function returns an `null` reference to `*Decoder`? We validate the object its not null before calling `Decode`

Reply:
Yes @Alevsk that is verified. When the body returns JSON that does not match the expected request schema, then the `Request` zero value will be used.

Also, the body is limited by a HTTP handler higher up the stack: https://github.com/minio/kes/blob/master/internal/http/handler.go#L74

See e.g. : https://go.dev/play/p/84asE1I6pez